manifests/cf-manifest/env-specific/prod.yml

```diff
@@ -8,3 +8,6 @@ scheduler_instances: 4
 cc_worker_instances: 4
 cc_hourly_rate_limit: 20000
 paas_region_name: ireland
+
+prometheus_disk_size: 500GB
+prometheus_retention_size: 475GB
```

Comment:
Should we keep the `prometheus_disk_size - prometheus_retention_size = 50GB` in both regions?

Reply:
This was 475GB before

I made 750/700 arbitrarily (roughly the same proportion)

Reply:
Ok. same proportion makes sense.